### PR TITLE
[Gabriela] Added Generics Task

### PR DESCRIPTION
**Pull request by gaenache**

### CODE REVIEW

**Review by abadarau — APPROVED** [review]

File: gabrielaEnache/src/main/java/com/endava/generics/GenericLibrary.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+public class GenericLibrary<T extends Media> {
+    private List<T> items;
+
+    public GenericLibrary() {
+        items = new ArrayList<>();
+    }
+
+    public void addItem(T item) {
+        if (items.size() < 6)
```
Comment (solonarieugeniu) [inline gabrielaEnache/src/main/java/com/endava/generics/GenericLibrary.java]: 6, magic number, please extract as a constant and assign a meaningful name

File: gabrielaEnache/src/main/java/com/endava/generics/Library.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    public void removeItem(Media item) {
+        this.items.remove(item);
+    }
+
+    public void clearItemList() {
+        this.items.clear();
+    }
+
+    public void removeMultipleItems(List<Media> list) {
```
Comment (solonarieugeniu) [inline gabrielaEnache/src/main/java/com/endava/generics/Library.java]: you remove all elements from list, not multiple, for multiple you should have a filter

File: gabrielaEnache/src/main/java/com/endava/generics/Video.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        else
+            System.out.println("Invalid duration value!");
+    }
+
+    public float getDuration() {
+        return duration;
+    }
+
+    public void setDuration(float duration) {
+        if (duration > 0)
```
Comment (solonarieugeniu) [inline gabrielaEnache/src/main/java/com/endava/generics/Video.java]: add brackets please, for organization and readability (all statements applicable)

File: gabrielaEnache/src/main/java/com/endava/generics/GenericLibrary.java (hunk 1 above)
Comment (gaenache) [reply gabrielaEnache/src/main/java/com/endava/generics/GenericLibrary.java]: Done

File: gabrielaEnache/src/main/java/com/endava/generics/Library.java (hunk 2 above)
Comment (gaenache) [reply gabrielaEnache/src/main/java/com/endava/generics/Library.java]: Done

File: gabrielaEnache/src/main/java/com/endava/generics/Video.java (hunk 3 above)
Comment (gaenache) [reply gabrielaEnache/src/main/java/com/endava/generics/Video.java]: Done

**Review by dvmunteanu — APPROVED** [review]

**Review by IIachimov — APPROVED** [review]

**Review by asecu — APPROVED** [review]

**Pull request merged**